packages/core/src/utils.js

```diff
@@ -1,4 +1,2 @@
 export { SAGA_ACTION, TASK } from '@redux-saga/symbols'
```

Comment:
the `redux-saga/utils` entry is starting to look useless, we should get rid of it later in favour of modular packages